### PR TITLE
Dev 20200429 change env var names

### PR DESCRIPTION
**Pull request by jpdorsch**
- Change all environment  variables adding prefix `F7T_`.
- Fixing typo in response header `X-Machine-Doesn-not-exists` to `X-Machine-Doesn-not-exist`

### CODE REVIEW

File: deploy/demo/common/common.env [hunk 1]
```diff
@@ -7,92 +7,92 @@
 # If AUTH_HEADER_NAME = Authorization, it can also check REALM_RSA_PUBLIC_KEY: RSA key from KeyCloak Realm which signs token.
 # REALM_RSA_PUBLIC_KEY="MII....QAB"
```
Comment (fcndl) [inline deploy/demo/common/common.env:8]: add prefix to env var names in comments

File: deploy/demo/common/common.env [hunk 2]
```diff
@@ -7,92 +7,92 @@
...
+F7T_KONG_URL=http://192.168.220.10:8000
 #-------
 # list of systems 
 #public name for systems, where users except to submit jobs and get files (list with ';')
-SYSTEMS_PUBLIC='cluster;cluster'
+F7T_SYSTEMS_PUBLIC='cluster;cluster'
 # filesystems mounted in each system
 # ; separated for system (position related to SYSTEMS_PUBLIC) and for each filesystem mounted inside each system, separated with ","
 # example: let's suppose SYSTEMS_PUBLIC="cluster1;cluster2", cluster1 has "/home" and "/scratch", and cluster2 has mounted "/home":
 # FILESYSTEMS = "/home,/scratch;/home"
```
Comment (fcndl) [inline deploy/demo/common/common.env:58]: add prefix to env var names in comments

File: deploy/test-build/docker-compose.yml [hunk 3]
```diff
@@ -11,7 +11,6 @@ services:
     volumes:
       - "./environment/keys/ca-key:/ca-key:ro"
       - "./environment/keys/user-key.pub:/user-key.pub:ro"
-      - ./logs/firecrest:/var/log:delegated
```
Comment (fcndl) [inline deploy/test-build/docker-compose.yml:14]: restore log volume declarations

Comment (jpdorsch) [reply deploy/test-build/docker-compose.yml]: Done

**Pull request merged**